### PR TITLE
[fix](planner) Fix  inconsistent nullability between outputTuple and groupByExpr when executing agg query

### PR DESCRIPTION
**Pull request by Kikyou1997**
# Proposed changes

Issue Number: #11360

## Checklist(Required)

1. Does it affect the original behavior: 
    - [x] Yes

2. Has unit tests been added:

    - [x] No

3. Has document been added or modified:

    - [x] No

4. Does it need to update dependencies:

    - [x] No
5. Are there any changes that cannot be rolled back:

    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — CHANGES_REQUESTED** [review]
Could you add a simple regression test case?

File: fe/fe-core/src/main/java/org/apache/doris/analysis/AggregateInfo.java [hunk 1]
```diff
@@ -464,6 +464,23 @@ public void substitute(ExprSubstitutionMap smap, Analyzer analyzer) {
         if (secondPhaseDistinctAggInfo != null) {
             secondPhaseDistinctAggInfo.substitute(smap, analyzer);
         }
+
+        for (SlotDescriptor slotDesc : getOutputTupleDesc().getSlots()) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/AggregateInfo.java:468]: please add some comments

**Comment by Kikyou1997** [comment]
> Could you add a simple regression test case?

done

File: fe/fe-core/src/main/java/org/apache/doris/analysis/AggregateInfo.java (hunk 1 above)
Comment (Kikyou1997) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/AggregateInfo.java:484]: done

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by morrySnow** [comment]
lgtm

**Review by morningman — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]

**Pull request merged**